src/import/import.c

```diff
@@ -471,6 +471,9 @@ static int parse_argv(int argc, char *argv[]) {
                         return log_error_errno(r, "Failed to pick image root: %m");
         }
 
+        if (arg_runtime_scope == RUNTIME_SCOPE_USER)
+                arg_import_flags |= IMPORT_FOREIGN_UID;
```

Comment:
So let's leave this for another PR, but to me it makes sense to use the foreign UID range for both system and user scope. The only reason to not do it for the system scope would be backwards compat? We should consider making it the default for the system scope at some point

Reply:
fully agree. but needs some strategy for compat